packages/terminal/src/common/shell-terminal-protocol.ts

```diff
@@ -25,7 +26,17 @@ export interface IShellTerminalServer extends IBaseTerminalServer {
 
 export const shellTerminalPath = '/services/shell-terminal';
 
+type shellTerminalOSPreferences<T> = {
```

Comment:
`export type ShellTerminalOSPreferences` - types should be named in CamelCase and since it is used in exposed type it should be exposed as well

Reply:
Done